htdocs/src/Oc/Language/LanguageRepository.php

```diff
@@ -39,9 +37,8 @@ public function __construct(Connection $connection)
     /**
      * Fetches all languages.
      *
-     * @return LanguageEntity[]
```

Comment:
this looks weird - why does php_cs sort return after throws?

Reply:
http://docs.phpdoc.org/references/phpdoc/tags/throws.html <-- i think this is a more "standard" way :)